triage/2022-04-05.md

```diff
@@ -0,0 +1,57 @@
+| | Regressions 😿 <br />(primary) | Regressions 😿 <br />(secondary) | Improvements 🎉 <br />(primary) | Improvements 🎉 <br />(secondary) | All 😿 🎉 <br />(primary) |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| count | 20 | 13 | 0 | 6 | 20 |
+| mean | 0.8% | 0.4% | N/A | -0.4% | 0.8% |
+| max | 2.0% | 0.7% | N/A | -0.5% | 2.0% |
+- It's not clear what the underlying cause could be as it seems that the issues are coming in trait selection and there are several PRs that touch trait selection (albeit for diagnostics purposes). I've left [a comment](https://github.com/rust-lang/rust/pull/95662#issuecomment-1088686029) to see if we can investigate further.
+
+#### Improvements
+
+Yet more `parse_tt` improvements [#95425](https://github.com/rust-lang/rust/pull/95425) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=bb5c437a2ce9ccf2204c974300c5ea9eb32d3635&end=c5cf08d37b85f953b132951e868df5b924250fdc&stat=instructions:u)
+| | Regressions 😿 <br />(primary) | Regressions 😿 <br />(secondary) | Improvements 🎉 <br />(primary) | Improvements 🎉 <br />(secondary) | All 😿 🎉 <br />(primary) |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| count | 1 | 0 | 30 | 23 | 31 |
+| mean | 0.3% | N/A | -0.9% | -2.0% | -0.9% |
+| max | 0.3% | N/A | -2.3% | -6.2% | -2.3% |
+
+
+Add debug assertions to some unsafe functions [#92686](https://github.com/rust-lang/rust/pull/92686) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=15a242a432c9c40a60def102209a5d40900b7b9d&end=168a0209002fef203e83989ff641c1b3e1a51859&stat=instructions:u)
+| | Regressions 😿 <br />(primary) | Regressions 😿 <br />(secondary) | Improvements 🎉 <br />(primary) | Improvements 🎉 <br />(secondary) | All 😿 🎉 <br />(primary) |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| count | 1 | 0 | 17 | 19 | 18 |
+| mean | 0.6% | N/A | -0.5% | -0.4% | -0.4% |
+| max | 0.6% | N/A | -1.4% | -0.8% | -1.4% |
+
+
+#### Mixed
+
+A new matcher representation for use in `parse_tt` [#95555](https://github.com/rust-lang/rust/pull/95555) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=d5139f44690e7765df801ca33a7f627d128ac9e2&end=6a9080b25e73d26aae94c3f6a13b31de58e66b5a&stat=instructions:u)
+| | Regressions 😿 <br />(primary) | Regressions 😿 <br />(secondary) | Improvements 🎉 <br />(primary) | Improvements 🎉 <br />(secondary) | All 😿 🎉 <br />(primary) |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| count | 3 | 10 | 18 | 11 | 21 |
+| mean | 0.4% | 1.0% | -1.0% | -2.4% | -0.8% |
+| max | 0.4% | 2.8% | -2.0% | -4.6% | -2.0% |
+- This was already [justified](https://github.com/rust-lang/rust/pull/95555#issuecomment-1088070142) as being acceptable since the real world crates that suffer from regressions are exercising some of the worst case scenario execution paths. There is an attempt to address this in a [follow up](https://github.com/rust-lang/rust/pull/95555#issuecomment-1088070142). In any case, the improvements outweigh the regressions in both magnitude and number.
```

Comment:
https://github.com/rust-lang/rust/pull/95669 is the follow-up, if you want to mention that explicitly.